src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -38,46 +52,77 @@ public async Task<CreateViewModel> GetCreateViewModel(CreateRequest request)
                 Sectors = cacheItem.Sectors,
                 JobRoles = cacheItem.JobRoles,
                 Levels = cacheItem.Levels,
+                LevelOptions = levels,
+                SectorOptions = sectors,
+                JobRoleOptions = jobRoles
```

Comment:
Linked to above comment this would now become:
```suggestion
                LevelOptions = levels.Result,
                SectorOptions = sectors.Result,
                JobRoleOptions = jobRoles.Result
```